.github/workflows/macos.yml

```diff
@@ -49,28 +51,44 @@ jobs:
         run: |
           mkdir _build
           echo "NPROC=`sysctl -n hw.ncpu`" >> $GITHUB_ENV
+          export TERM="vt100"
+          echo "TERM=$TERM" >> $GITHUB_ENV
 
       - name: configure
         run: |
           cd _build
-          ../configure --enable-cobc-internal-checks --enable-hardening
+          ../configure --enable-cobc-internal-checks --enable-debug --prefix /opt/cobol/gnucobol-gcos --exec-prefix /opt/cobol/gnucobol-gcos
```

Comment:
Is there a reason to drop the hardening? If not this should really stay in for the CI builder as it does catches some coding issues from time to time.

Reply:
I replaced `--enable-debug` by `--enable-hardening`, but maybe it is a good idea to have both ?

Reply:
The first _can_ be useful to get some ore insights, it is fine to add both.